sdcm/tester.py

```diff
@@ -1195,7 +1194,7 @@ def get_stress_results(self, queue, store_results=True):
         return results
 
     def get_stress_results_bench(self, queue):
-        results = self.loaders.get_stress_results_bench(queue)
+        results = queue.get_stress_results_bench()
```

Comment:
??? looks strange

Reply:
the name is misleading, but as per our conversation, it is used all over the code. So let's keep it as is...